### PR TITLE
Update sia-ui to 1.4.1.2

### PR DESCRIPTION
**Pull request by ianlevesque**
The project moved to GitLab at some point as well

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].
- [x] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

File: Casks/sia-ui.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
 cask 'sia-ui' do
-  version '1.3.7'
-  sha256 '759da9bb4df6744b6d90884f2e52bbcb078940ceae22b0c1ffc4f8e410d56f0f'
+  version '1.4.1.1'
+  sha256 '541405884d119ba394e073c54462ff378770d7cb8a69cec8aa475130ef8e0522'
 
-  # github.com/NebulousLabs/Sia-UI was verified as official when first introduced to the cask
-  url "https://github.com/NebulousLabs/Sia-UI/releases/download/v#{version}/Sia-UI-v#{version}-osx-x64.zip"
-  appcast 'https://github.com/NebulousLabs/Sia-UI/releases.atom'
```
Comment (vitorgalvao) [inline Casks/sia-ui.rb]: Please try to find a new appcast.

Comment (ianlevesque) [reply Casks/sia-ui.rb]: It looks like this came from github itself with the old hosting location, the developer doesn't use an appcast for anything nor provide one.

Comment (vitorgalvao) [reply Casks/sia-ui.rb]: > the developer doesn't use an appcast for anything nor provide one

Any page with the version number suffices, it doesn’t need to be provided by the developer. If the download is versioned, there must be something we can use.

Comment (ianlevesque) [reply Casks/sia-ui.rb]: I don't think I understand. I thought the appcast was required to be an RSS feed.

Comment (vitorgalvao) [reply Casks/sia-ui.rb]: > I thought the appcast was required to be an RSS feed.

[It’s not](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md):

> An appcast can be any URL hosted by the app’s developer that changes every time a new release is out or that contains the version number of the current release (e.g. a download HTML page). Webpages that only change on new version releases are preferred, as are sites that do not contain previous version strings (i.e. avoid changelog pages if the download page contains the current version number but not older ones).

**Pull request merged**